raiden/raiden_event_handler.py

```diff
@@ -603,104 +603,102 @@ def handle_contract_send_channelunlock(
     def handle_contract_send_channelsettle(
         raiden: "RaidenService", channel_settle_event: ContractSendChannelSettle
     ):
-        # only make settlements if our node is a participant, dont do it for lc
-        if channel_settle_event.channel_state.our_state.address == raiden.address or channel_settle_event.channel_state.partner_state.address == raiden.address:
-            assert raiden.wal, "The Raiden Service must be initialize to handle events"
-            canonical_identifier = CanonicalIdentifier(
-                chain_identifier=raiden.chain.network_id,
-                token_network_address=channel_settle_event.token_network_identifier,
-                channel_identifier=channel_settle_event.channel_identifier,
+        assert raiden.wal, "The Raiden Service must be initialize to handle events"
```

Comment:
are you sure that by checking `wal != None` we can assert that the raiden service is initialized?

(also: typo on `initialized`)